ee/clickhouse/queries/experiments/funnel_experiment_result.py

```diff
@@ -144,8 +144,8 @@ def calculate_results(
         if not control_variant:
             raise ValidationError("No control variant data found", code="no_data")
 
-        if len(test_variants) > 3:
-            raise ValidationError("Can't calculate A/B test results for more than 4 variants", code="too_much_data")
+        if len(test_variants) >= 10:
```

Comment:
How much slower will this cause things to be?

Reply:
Not as bad as I was expecting, since we've switched how we compute results to be a monte-carlo simulation vs a closed-form solution I was trying before! (this happened during initial implementation, I just forgot that it means the limit of 3 isn't necessary anymore)

TL;DR -> It's almost no difference to before, as the limiting factor here is getting the insight results, not this computation.